From c5cbcef415b16a8d0168c0d6b9da467d8c4e59a3 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Thu, 6 Feb 2020 16:01:39 +0800
Subject: [PATCH] linker.cpp: fix for missing RelocationInfo.h

Facing build failure with below error:

git/core/compiler_interface/linker.cpp:14:10: fatal error: RelocationInfo.h: No such file or directory
|    14 | #include "RelocationInfo.h"
|       |          ^~~~~~~~~~~~~~~~~~
| compilation terminated.

Upstream-Status: Submitted [https://github.com/intel/compute-runtime/pull/256]

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 core/compiler_interface/linker.cpp                  | 2 +-
 core/unit_tests/compiler_interface/linker_tests.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/compiler_interface/linker.cpp b/core/compiler_interface/linker.cpp
index a5b0386e9..c1445b336 100644
--- a/core/compiler_interface/linker.cpp
+++ b/core/compiler_interface/linker.cpp
@@ -11,7 +11,7 @@
 #include "core/helpers/debug_helpers.h"
 #include "core/helpers/ptr_math.h"
 
-#include "RelocationInfo.h"
+#include "visa/RelocationInfo.h"
 
 #include <sstream>
 
diff --git a/core/unit_tests/compiler_interface/linker_tests.cpp b/core/unit_tests/compiler_interface/linker_tests.cpp
index 6bb7e987c..f2d0b5632 100644
--- a/core/unit_tests/compiler_interface/linker_tests.cpp
+++ b/core/unit_tests/compiler_interface/linker_tests.cpp
@@ -11,7 +11,7 @@
 
 #include <string>
 
-#include "RelocationInfo.h"
+#include "visa/RelocationInfo.h"
 
 TEST(LinkerInputTests, givenGlobalsSymbolTableThenProperlyDecodesGlobalVariablesAndGlobalConstants) {
     NEO::LinkerInput linkerInput;
-- 
2.17.1

